lib/fizzy/execute.cpp

```diff
@@ -356,7 +388,7 @@ inline T ffloor(T value) noexcept
     // the __builtin_floor() outputs -0 where it should +0.
     // The following workarounds the issue by using the fact that the sign of
     // the output must always match the sign of the input value.
-    return std::copysign(result, value);
+    return fcopysign(result, value);
```

Comment:
This and the other cases may be faster or may be slower if the rest of the function already uses SSE registers.